www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Next generation web applications with async PHP
+info: https://phpconference.com/php-core-coding/next-generation-web-applications-with-async-php/
+location: Berlin
+tags:
+  - next generation web applications with async php 
```

Comment:
Not sure about a proper tag, but should probably be shorter and dash separated like other tags. Any thoughts?

Reply:
Tag shortened sensibly and dashes added